assets/src/js/app.js

```diff
@@ -9,6 +9,7 @@ import {setupExternalLinks} from './external_links';
 import {setupListingPages} from './listing_pages';
 import {setupQueryLoopCarousel} from './query_loop_carousel';
 import {setupClickabelActionsListCards} from './actions_list_clickable_cards';
+import {removeNoPostText} from './query-no-posts';
```

Comment:
What do you think if we move this function into [the listing pages setup file](https://github.com/greenpeace/planet4-master-theme/blob/15a9ccf5fdb89933dbda704fb081d2e3337ab786/assets/src/js/listing_pages.js)?

Reply:
Oh sorry! I shared the wrong file 🙏  I meant into the [Query Loop extension](https://github.com/greenpeace/planet4-master-theme/blob/c1c59752ca53594edb804b59d29a0327bac89b83/assets/src/block-editor/QueryLoopBlockExtension/index.js).
Do you think it's better?

Reply:
Oh yea great suggestion, will update the code asap

Reply:
Got it! Another suggestion and looking forward your point of view.
I was thinking something like this 👇 

```
register_block_type( 'core/query', array(
    'render_callback' => function (array $attributes, $content, $block) {
       // Here override the rendered content if the data is empty 
        return $content;
    },
) );
```

Does it make sense for you?

Reply:
It may work 🤔, but what about the case where editors may want to render some custom text or block since they know that is available with the core block, how do we handle that?